### ISSUE TITLE
Use-STSRoleWithSAML does not honor Set-AWSProxy

### ISSUE THREAD

**Issue by jgardsentry** [aws/aws-tools-for-powershell#7]
**Expected Behavior**
After you've run Set-AWSProxy, _all_ cmdlets from AWSPowershell should use that configuration unless explicitly bypassed.  Most do, Use-STSRoleWithSAML appears not to.

**Current Behavior**
Wireshark confirms traffic resulting from Use-STSRoleWithSAML attempting to go directly to the STS public endpoint IP address instead of utilizing proxy.  
Use-STSRoleWithSAML _does_ use the Internet Explorer proxy, if configured.  But this is inconsistent with the rest of the module, and not how we want to configure our servers.

**Steps to Reproduce (for bugs)**
(assume variables below like $SAMLAssertion are valid from an ADFS interaction.  I've done a lot with this stuff, and it works when IE proxy is configured)
```powershell
Set-AWSProxy -Hostname 'webproxy.company.com' -Port 80
Use-STSRoleWithSAML -SAMLAssertion $SAMLAssertion -RoleArn $RoleArn -PrincipalArn $PrincipalArn -DurationInSeconds 28800 -region us-east-1
```
...will timeout when banging against our firewall in the default route.  Setting the IE proxy to the same webproxy.company.com:80 makes it work immediately.

**Context**
In an enterprise with required outbound web proxy.  Trying to configure an automation/orchestration tool to get temporary AWS credentials from ADFS.  This routine works from my workstation because I have the IE proxy configured.  We don't often configure that on servers.

**Your Environment**
* Affected module: AWSPowerShell
* Module version as reported by `Get-AWSPowerShellVersion`: 
> AWS Tools for Windows PowerShell
> Version 3.3.498.0
> Copyright 2012-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
> 
> Amazon Web Services SDK for .NET
> Core Runtime Version 3.3.101.0
> Copyright 2009-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.

* Operating System and version as reported by `$PSVersionTable.OS`: Windows Server 2016 v1607
* PowerShell versions as reported by `$PSVersionTable.PSEdition`: Desktop 5.1.14393.2879

**Comment by matteo-prosperi** [comment]
I am looking into this issue. We will include a fix in the next update of the modules.

**Comment by jgardsentry** [comment]
Thanks Matteo.  It looks like this module has a release roughly every month.  Is that looking like the case for the current release?  ~end of May?

**Comment by matteo-prosperi** [comment]
We expect to release an update of the AWSPowerShell modules this week.

**Comment by matteo-prosperi** [comment]
We released 3.3.509 today, this problem should be fixed. Please reopen the issue in case of any problem.
